### PR TITLE
Fix typo in CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by jasmussen**
Teensy typo fix. Going to merge when the checks clear.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ This list is manually curated to include valuable contributions by volunteers th
 | @ameeker | |
 | @StaggerLeee | |
 | @jblz | |
-| | @nicbertino |
+| | @nic-bertino |
```
Comment (gziolo) [inline CONTRIBUTORS.md]: 2nd column is WP.org profile. I guess it was okey: https://profiles.wordpress.org/nicbertino.

Comment (jasmussen) [reply CONTRIBUTORS.md]: Raaah, hah, will fix, thanks!

**Review by gziolo — APPROVED** [review]
:shipit:

**Comment by jasmussen** [comment]
❤️

**Pull request merged**